docs/docs/chrome-extension/features.md

```
@@ -4,34 +4,34 @@
 
 <img src="https://khulnasoft.com/images/pr_insight/pr_chat_1.png" width="768">
 <img src="https://khulnasoft.com/images/pr_insight/pr_chat_2.png" width="768">
 
 
 ### Toolbar extension
-With PR-Insight Chrome extension, it's [easier than ever](https://www.youtube.com/watch?v=gT5tli7X4H4) to interactively configure and experiment with the different tools and configuration options.
+With Khulnasoft Merge Chrome extension, it's [easier than ever](https://www.youtube.com/watch?v=gT5tli7X4H4) to interactively configure and experiment with the different tools and configuration options.
 
 For private repositories, after you found the setup that works for you, you can also easily export it as a persistent configuration file, and use it for automatic commands.
 
 <img src="https://khulnasoft.com/images/pr_insight/toolbar1.png" width="512">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)